HashLib4CSharp/src/Checksum/Adler32.cs

```diff
@@ -10,7 +10,10 @@ HashLib4CSharp Library
 This library was sponsored by Sphere 10 Software (https://www.sphere10.com)
 for the purposes of supporting the XXX (https://YYY) project.
 */
+<<<<<<< HEAD
 
+=======
+>>>>>>> 26569c7... Improve Adler32 performance
```

Comment:
@azureskydiver , first of all, thanks for this pull request.
for some reasons, it seems your git commit added some unneeded info to the source which caused the travis build to fail.
the affected line are 13 - 16.
can you please resolve this so I can merge the pull request?
Thanks.

Reply:
I know. I couldn't figure out why it things there is a merge conflict when what I did was just delete a blank line.

Anyway, I put in a new request hopefully without the merge conflict.